pkg/reconciliation/reconcile_racks.go

```diff
@@ -248,6 +241,11 @@ func (rc *ReconciliationContext) CheckVolumeClaimSizes(statefulSet, desiredSts *
 				return result.Error(fmt.Errorf(msg))
 			}
 
+			supportsExpansion, err := rc.storageExpansion()
+			if err != nil {
+				return result.Error(err)
+			}
+
```

Comment:
[question] Are we moving this inside the loop on the assumption that volume expansions will be rare?

Reply:
Yes and this check is irrelevant in our other use cases right now.